test/parallel/test-cluster-worker-exit.js

```diff
@@ -106,17 +106,13 @@ function checkResults(expected_results, results) {
     var actual = results[k],
         expected = expected_results[k];
 
-    if (typeof expected === 'function') {
-      expected(r[k]);
```

Comment:
This is an actual error with the undefined `r` variable here. Because the condition was never met, I removed the whole `if` block.